features/list_companies_of_category.feature

```diff
@@ -0,0 +1,36 @@
+Feature: As a visitor
+  In order to easily find a member of a certain category
```

Comment:
Use the word 'company' instead of 'member'; "member" = a person that is a member of SHF.